src/pages/MentorProfile/MentorProfile.component.tsx

```diff
@@ -229,15 +241,6 @@ const MentorProfile: React.FC = () => {
             </div>
           </div>
         )}
-      <InformationModal
-        isOpen={isOpen}
-        headline="Withdraw your current application"
-        body={`Are you sure you want to withdraw your current application and apply for ${
-          mentor?.application.firstName ?? ''
-        } ${mentor?.application.lastName ?? ''}?`}
-        onConfirm={revokeApplication}
-        onClose={closeModal}
-      />
```

Comment:
Why this is removed? @AnsarMahir

Reply:
@anjula-sack had to go back to an old commit due to removing the mentors profession. Copied that code and this was from a latest one so it also gone. Let me fix it this by today